visualization/app/codeCharta/ui/copyToClipboardButton/util/getFilenamesWithHighestMetric.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { NodeType } from "../../../codeCharta.model"
+import { FileToValue, getFilenamesWithHighestMetrics, updateAttributeMap } from "./getFilenamesWithHighestMetrics"
+
+let MAP: Map<string, FileToValue[]>
```

Comment:
The parsing logic behaves kindly and doesn't mutate any passed in state, does it? So we can and should make it a const.

Reply:
Actually, the method `updateAttributeMap` mutates `MAP` in its test suites.